code/examples/preact-kitchen-sink/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "jsx": "react-jsx",
+    "jsxFactory": "h",
+    "jsxFragmentFactory": "Fragment",
```

Comment:
This seems to give me error/warnings when I open the file:
![Screen Shot 2022-07-27 at 11 31 24](https://user-images.githubusercontent.com/3070389/181213908-e58b957e-f0d7-4b5c-a62a-ea2f49263721.png)